### PR TITLE
Updatespons

### PR DESCRIPTION
**Pull request by AdityaLawate**
changed sponsors page and media page

### CODE REVIEW

File: pages/index.html
Comment (sharvin18) [inline pages/index.html:1]: Why is the same change reflecting in 2 different branches and pull requests?

File: images/yt.svg
Comment (sharvin18) [inline images/yt.svg:1]: Why is the same change reflecting in 2 different branches and pull requests?

**Review by sharvin18 — CHANGES_REQUESTED** [review]
Same changes in different branches.
Be careful when you pull from main into your branch.
If changes not needed, just delete them

File: pages/index.html
Comment (AdityaLawate) [reply pages/index.html:1]: 1. I dont know why the same changes are reflecting again and again
2. How to delete the changes?

**Pull request closed without merging**